controllers/downloads.js

```diff
@@ -91,16 +97,21 @@ var getSumOfDays = function(request,reply,conditions,period) {
               error: "no stats for this package for this period (0002)"
             })
           } else {
-            var result = rows[0]
-            var output = {
-              downloads: result.downloads,
-              start: period.start,
-              end: period.end
-            }
-            if (!conditions.all) {
-              output.package = result.package
-            }
-            reply(output)
+            var outputs = []
```

Comment:
Same comment as handleBulkRange; it would be more convenient if output were an object indexed by package name.